{"name": "test_authorization[data1]", "status": "failed", "statusDetails": {"message": "AssertionError: assert None\n +  where None = <bound method BaseSetup.check_if_element_exists of <Base.base.BaseSetup object at 0x00000244FDEB3EC8>>(('css selector', 'nav ul.list-unstyled .sidebar-header:nth-child(2)'))\n +    where <bound method BaseSetup.check_if_element_exists of <Base.base.BaseSetup object at 0x00000244FDEB3EC8>> = <Base.base.BaseSetup object at 0x00000244FDEB3EC8>.check_if_element_exists\n +      where <Base.base.BaseSetup object at 0x00000244FDEB3EC8> = <utilities.testFrame.InitPages object at 0x00000244FDEB3648>.base\n +    and   ('css selector', 'nav ul.list-unstyled .sidebar-header:nth-child(2)') = locator.PROFILE", "trace": "app = <utilities.testFrame.InitPages object at 0x00000244FDEB3648>, data = ['admin@gmail.com', 'qaz1qaz']\n\n    @allure.link(\"http://localhost:50621/home/events?page=1\", name='Click me')\n    @allure.feature('Login User')\n    @allure.story('\"Actors\" login to site EventExpress ')\n    @allure.severity(allure.severity_level.CRITICAL)\n    @pytest.mark.parametrize('data',credentials())\n    def test_authorization(app,data):\n        app.auth.click_on_login_button()\n        app.auth.clean_login_field( )\n        app.auth.type_login(data[0])\n        app.auth.clean_password_field( )\n        app.auth.type_pass(data[1])\n        app.auth.press_button_signin()\n>       assert app.base.check_if_element_exists( locator.PROFILE )\nE       AssertionError: assert None\nE        +  where None = <bound method BaseSetup.check_if_element_exists of <Base.base.BaseSetup object at 0x00000244FDEB3EC8>>(('css selector', 'nav ul.list-unstyled .sidebar-header:nth-child(2)'))\nE        +    where <bound method BaseSetup.check_if_element_exists of <Base.base.BaseSetup object at 0x00000244FDEB3EC8>> = <Base.base.BaseSetup object at 0x00000244FDEB3EC8>.check_if_element_exists\nE        +      where <Base.base.BaseSetup object at 0x00000244FDEB3EC8> = <utilities.testFrame.InitPages object at 0x00000244FDEB3648>.base\nE        +    and   ('css selector', 'nav ul.list-unstyled .sidebar-header:nth-child(2)') = locator.PROFILE\n\ntest_auth.py:28: AssertionError"}, "attachments": [{"name": "stdout", "source": "dc95b24d-6151-4840-9cd8-aeba8bede835-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data", "value": "['admin@gmail.com', 'qaz1qaz']"}], "start": 1577191903356, "stop": 1577191915206, "uuid": "73c0ac3c-6716-41cd-8d79-67ac29e3978e", "historyId": "307b3e81a3c9742e74203e6924c08bb2", "testCaseId": "2a55ae64119e6f3ed897e368050f6f6b", "fullName": "Tests.test_auth#test_authorization", "labels": [{"name": "severity", "value": "critical"}, {"name": "feature", "value": "Login User"}, {"name": "story", "value": "\"Actors\" login to site EventExpress "}, {"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_auth"}, {"name": "host", "value": "DESKTOP-C599ACF"}, {"name": "thread", "value": "7364-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_auth"}], "links": [{"type": "link", "url": "http://localhost:50621/home/events?page=1", "name": "Click me"}]}